content/applications/general/calendars/google/google_calendar_credentials.rst

```diff
@@ -45,58 +46,57 @@ Calendar API` from the search results. Click :guilabel:`Enable`.
-credentials. Begin by clicking :guilabel:`Create Credentials`. Google will then guide you through
-four steps to create your API credentials.
+Now that the API project has been created OAuth should be configured. Click on :guilabel:`OAuth
+consent` in the left menu and then select the :guilabel:`User Type`.
 
-- In the first step, :guilabel:`Credential Type`, select the :guilabel:`Google Calendar API` and
-  :guilabel:`User Data` options. Then, click :guilabel:`Next`.
+.. warning::
+   Personal Gmail Accounts are only allowed to be :guilabel:`External` User Type, which means
+   Google may require an approval or for :guilabel:`Scopes` to be added on. Using a Google WorkSpace
+   account allows for :guilabel:`Internal` User Type. It should also be noted that while the API
+   connection is in the :guilabel:`External` :guilabel:`Testing` mode then no approval is necessary
+   from Google. User limits in this :guilabel:`Testing` mode is set to 100 users.
 
-.. image:: google_calendar_credentials/credential-type.png
-   :align: center
-   :alt: Select Google Calendar API and User Data for the Credential Type.
+In the second step, :guilabel:`OAuth Consent Screen`, type *Odoo* in the :guilabel:`App name`
+field, select the email address for the :guilabel:`User support email` field, and type the email
+address for the :guilabel:`Developer contact information` section. Then, click :guilabel:`Save
+and Continue`.
+
+Skip the third step, :guilabel:`Scopes`, by clicking :guilabel:`Save and Continue`.
+
+Next, if continuing in testing mode (External), add the email addresses being configured under the
+:guilabel:`Test users` step by clicking on :guilabel:`Add Users` and then the :guilabel:`Save and
+Continue` button. A summary of the :guilabel:`App registration` appears.
 
-- In the second step, :guilabel:`OAuth Consent Screen`, type *Odoo* in the :guilabel:`App name`
-  field, select your email address for the :guilabel:`User support email` field, and type your email
-  address for the :guilabel:`Developer contact information` section. Then, click :guilabel:`Save
-  and Continue`.
+Finally, scroll to the bottom and click on :guilabel:`Back to Dashboard`.
 
-- Skip the third step, :guilabel:`Scopes`, by clicking :guilabel:`Save and Continue`.
+Now the OAuth consent has been configured and it's time to create credentials.
+
+Create credentials
+------------------
 
-- In the last step, :guilabel:`OAuth Client ID`, select :guilabel:`Website application` for the
-  :guilabel:`Application Type` field and type *My Odoo Database* for the :guilabel:`Name`.
+The **Client ID** and the **Client Secret** are both needed to connect Google Calendar to Odoo. This
+is the last step in the Google console, begin by clicking :guilabel:`Credentials` in the left menu.
+Then click :guilabel:`Create Credentials` and select :guilabel:`OAuth client ID` Google will open a
+guide to create credentials.
```

Comment:
The **Client ID** and the **Client Secret** are both needed to connect Google Calendar to Odoo. This
is the last step in the Google console. Begin by clicking :guilabel:`Credentials` in the left menu.
Then, click :guilabel:`Create Credentials` , select :guilabel:`OAuth client ID`, and Google will open a
guide to create credentials.